forms/TransportGoodsForm.tsx

```diff
@@ -153,44 +188,64 @@ export const TransportGoodsForm = ({ onSubmit }: ITransportGoodsFormProps) => {
             <Radio
               value={typeOptions && typeOptions[0]?.value}
               {...register('transportType')}
+              className={clsx('mb-4')}
             >
               {typeOptions && typeOptions[0]?.display_name}
             </Radio>
-            <Radio value={typeOptions && typeOptions[1]?.value} {...register('transportType')} className={clsx('mb-0')}>
-              <Dropdown {...register('transportCounty')}
+            <Radio
+              value={typeOptions && typeOptions[1]?.value}
+              {...register('transportType')}
+              className={clsx('mb-0')}
+            >
+              <Dropdown
+                {...register('transportCounty')}
                 disabled={watchTransportType !== TransportType.County}
                 errors={errors.transportCounty}
                 placeholder={t('services.county.placeholder')}
               >
                 {countiesOptions}
               </Dropdown>
             </Radio>
+            {/* 
+                // TODO Remove this after testing DropdownMultiSelect
+              */}
+            {/* <DropdownMultiSelect
```

Comment:
shouldn't this be the other way around? The Radio should include the control

Reply:
no, the radio will contain the choices for `type` and if `type` is County